integration-tests/micrometer/src/test/java/org/apache/camel/quarkus/component/micrometer/it/MicrometerTest.java

```diff
@@ -83,20 +84,58 @@ public void testMicrometerExchangeEventNotifier() {
         assertTrue(getMetricValue(Double.class, "timer", "CamelExchangeEventNotifier", tags) >= 0.0);
     }
 
+    @Test
+    public void testAnnotations() {
+        RestAssured.get("/micrometer/annotations/call")
+                .then()
+                .statusCode(200);
+        RestAssured.get("/micrometer/annotations/call")
+                .then()
+                .statusCode(200);
+
+        int count = RestAssured.get("/micrometer/annotations/get")
```

Comment:
@aldettinger I have to get count from the object, because it can have value 2 or 3. Depends whether testMetricsApiRenamed was executed or not.